Brokkr/src/main/cpp/subsystems/Arm.cpp

```diff
@@ -14,10 +14,11 @@ namespace
     }
 }
 
-Arm::Arm(WPI_TalonFX& mArmM1, WPI_TalonFX& mArmM2, WPI_TalonFX& mTeleArm)
+Arm::Arm(WPI_TalonFX& mArmM1, WPI_TalonFX& mArmM2, WPI_TalonFX& mTeleArm, WPI_CANCoder& mCanCoder)
 : ArmMotor1(mArmM1)
 , ArmMotor2(mArmM2)
 , TelescopingArm(mTeleArm)
+, CanCoder(mCanCoder)
```

Comment:
Use lower case for parameter names. The `m` prefix is reserved for member variable names.

```
Arm::Arm(WPI_TalonFX& armM1, WPI_TalonFX& armM2, WPI_TalonFX& teleArm, WPI_CANCoder& canCoder)
: mArmMotor1(armM1)
, mArmMotor2(armM2)
, mTelescopingArm(teleArm)
, mCanCoder(canCoder)
```

Lots of other changes are needed in the file to follow the naming convention.

Reply:
Fix later